apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -766,10 +788,10 @@ const BookingPage = ({
                       rows={3}
                       className={classNames(
                         "focus:border-brand block w-full rounded-sm border-gray-300 shadow-sm focus:ring-black dark:border-gray-900 dark:bg-gray-700 dark:text-white dark:selection:bg-green-500 sm:text-sm",
-                        disableInput ? "bg-gray-200 dark:text-gray-500" : ""
+                        disableInput && loggedInIsOwner ? "bg-gray-200 dark:text-gray-500" : ""
                       )}
                       placeholder={t("share_additional_notes")}
-                      disabled={disableInput}
+                      disabled={disableInput && loggedInIsOwner}
```

Comment:
This is causing the tests to fail as it expects for the field to be disabled and it is enabled for a logged out user. Is this intended?